content/plugin/log/filtering.mdx

```diff
@@ -0,0 +1,73 @@
+---
+page_title: 'Home - Plugin Development: Filtering Log Output'
+description: >-
+  Information on how to filter the logs that your provider outputs, to help
+  find the information you're looking for.
+---
+
+# Filtering Log Output
+
+A lot of different pieces of software are included in a Terraform run: Terraform itself, the SDK or SDKs that providers are built on, and one or more providers. All of these disparate pieces of software can be writing logs, which means that a lot of different logs are being written, and not all of them are always useful in figuring out why something's happening. To help with that, there are various environment variables used to filter log output.
```

Comment:
Can we say something up here about how this page is going to list out the environment variables for every distinct thing that is active in the Terraform run and how you can choose how verbose you want the output to be from each of these things (by setting a level).

Also - can we mention the SDK vs. the framework here? I feel like reading this I'm not sure which one (both?) this page is going to apply to.

I just want to set the page up a bit better so users understand what they're getting and what's going on below.